ansible/userdata/cloud-config-etcd

```diff
@@ -40,6 +40,36 @@ coreos:
   update:
     reboot-strategy: "off"
   units:
+    - name: al-agent-container.service
+      command: start
+      enable: true
+      content: |
+        [Unit]
+        Description=Alert Logic Container Service
+        After=docker.service
+        Requires=docker.service
+        [Service]
+        TimeoutStartSec=0
+        Restart=always
+        ExecStartPre=-/usr/bin/docker stop al-agent-container
+        ExecStartPre=-/usr/bin/docker rm al-agent-container
+        ExecStartPre=/usr/bin/docker pull alertlogic/al-agent-container:latest
+        ExecStart=/usr/bin/docker run \
+          --name=al-agent-container \
+          --label "app=al-agent-container" \
+          --net=host \
+          --privileged \
+          --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
+          --mount type=bind,source=/proc,target=/host/proc \
+          --memory-reservation=100m \
+          --memory=500m \
+          --cpus=1 \
+          --restart=always \
+          alertlogic/al-agent-container:latest
```

Comment:
Still no versioning?

Reply:
https://github.com/Financial-Times/content-alertlogic-helm-config/blob/master/helm/alert-logic/values.yaml#L8 - we are using the latest version of alert logic for our daemonsets

Reply:
I know, but the questions still remains - for all other services  we use versioned images so we are in control of what is running in our cluster.

Reply:
Could not really see any Dockerfile. Does not really tell me much about what the image has: https://github.com/alertlogic/al-agent-container/tree/master/docker & https://hub.docker.com/r/alertlogic/al-agent-container/.

It is a intrusion detection service. I guess they want to keep things secret -_-